.github/workflows/ci-web.yml

```diff
@@ -63,6 +63,11 @@ jobs:
         working-directory: client
         timeout-minutes: 5
 
+      - name: Check prettier
+        run: npx prettier . --check
```

Comment:
This is different than `pre-commit run prettier --all-files`, the later command would include the javascript files under `bindings`.

Reply:
Yeah, I removed `bindings` from the pre-commit. Let's keep it simple with only the client folder for now.

Reply:
Will likely be forgotten when we add bindings.

Reply:
Yeah but it's all auto-generated stuff or boilerplate that will never change. I would have to move prettier config from the client folder to the root folder, and if prettier finds faults with the generated code, I would also have to change the template. I can open an issue if we really want prettier on the code in bindings, but honestly, it's probably just wasted time.